examples/example_config_files/public_TrueCoaxial.yaml

```diff
@@ -26,23 +26,25 @@ medium: vacuum
 detectors:
 - semiconductor:
     material: HPGe
-    temperature: 77
+    temperature: 90
     impurity_density:
-      name: constant
-      value: 1e8cm^-3
+      name: PtypePNjunction
+      Li_annealing_temperature: 623
+      Li_annealing_time: 1080
```

Comment:
We never define units of time and it is not clear, whether this should be seconds, minutes, hours, ...
Maybe force this value to have a unit? Otherwise, we have to add another field to the `units` section in the config file and define our default unit for time in the config file.